src/BenchmarkDotNet.Core/Diagnosers/CompositeDiagnoser.cs

```diff
@@ -19,13 +19,13 @@ public CompositeDiagnoser(params IDiagnoser[] diagnosers)
 
         public IColumnProvider GetColumnProvider() 
             => new CompositeColumnProvider(diagnosers.Select(d => d.GetColumnProvider()).ToArray());
-
+#if !UAP
         public void BeforeAnythingElse(Process process, Benchmark benchmark) 
```

Comment:
There is System.Diagnostics.Process nuget package. But when I add ```runtimes``` section to a project where the package is added, tooling starts complaining about it.